CONTRIBUTING.md

```diff
@@ -1,28 +1,35 @@
- # Contributing to Mongock project
+# Contributing to Mongock project
 
 :tada: First off, thanks for taking the time to contribute! :tada:
 
  The following is a set of guidelines for contributing to Mongock project which is hosted on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel
  free to propose changes to this document in a pull request.
 
 
-## Getting started
+## Getting started :rocket:
 
-> :boom::rocket: Visit our page of [requested contribution](https://github.com/mongock/mongock/labels/contribution-requested) to find out the best candidates and which of them are rewarded!!
+> :boom: Visit our page of [requested contribution](https://github.com/mongock/mongock/labels/contribution-requested) to find out the best candidates and which of them are rewarded!!
 
-Contributions are made to this repo via **issues**, **discussions** and **pull request**.
+Contributions are made to this repo via **issues**, **discussions** and **code contributions**.
 
-Within the Pull Request section, there is a special kind of contribution that deserves special attention:  [Creation of new drivers](#providing-a-new-driver)
-
-
-## Issues
+## Issues :bug:
 For raising a new issue, please visit this [link](https://github.com/mongock/mongock/issues/new?assignees=&labels=&template=bug_report.md&title=), fills the template and submit it. 
 You will hear from us shortly.
 
-## Discussions
-For general **Q&A**s, **Ideas**, **comment on any release** or anything you want to raise, but it's not an issue, please visit the [discussion page](https://github.com/mongock/mongock/discussions)
+**Contributor raising a valid issue(or solving it) will get listed in our [contributor list](https://github.com/mongock/mongock/blob/master/CONTRIBUTORS.md).**
```

Comment:
Can we perhaps manage the contributors list as:
- Collaborators (one-off contributions)
- Contributors (code maintainers) 
We can also have a separate definition - just an idea :)

Reply:
I agree. Somewhere ther I mention "external collaboration" to make that difference. We may need to work in separate those two concepts.